src/AddIns/Uno.UI.Lottie/LottieVisualSource.Android.cs

```diff
@@ -29,7 +29,7 @@ public void Update(AnimatedVisualPlayer player)
 			{
 				_animation = new LottieAnimationView(Android.App.Application.Context);
 				_animation.EnableMergePathsForKitKatAndAbove(true);
-				_animation.UseHardwareAcceleration(UseHardwareAcceleration);
+				_animation.SetRenderMode(RenderMode.Hardware);
```

Comment:
Is it related to Android 10X ?

Reply:
this is due to the new lottie version 3.0.4 that support Android X